app/experimenter/experiments/models.py

```diff
@@ -444,6 +444,12 @@ def _transition_date(self, old_status, new_status):
             if change.old_status == old_status and change.new_status == new_status:
                 return change.changed_on.date()
 
+    @property
+    def enrollment_complete_date(self):
+        change = self.changes.filter(message="Enrollment Complete")
```

Comment:
We gotta find a better way to deal with this case...

Filed a followup: https://github.com/mozilla/experimenter/issues/2798

Reply:
Oh oh, no make this behave the same as the others, if there's a real enrollment complete date return that otherwise return the computed one

Reply:
Oh you already have that below:

https://github.com/mozilla/experimenter/pull/2794/files#diff-562e6046de58c57f7e5e7f6ecdb15efbR486-R488

Just use that everywhere?

Reply:
relooking at what's already there, i think it makes sense to fold it into `enrollment_end_date`
I made changes around this, i think the time math still makes sense...